manim/__main__.py

```diff
@@ -75,26 +76,28 @@ def main():
 
     else:
         config.digest_args(args)
-
-        module = get_module(config.get_dir("input_file"))
-        all_scene_classes = get_scene_classes_from_module(module)
-        scene_classes_to_render = get_scenes_to_render(all_scene_classes)
-        for SceneClass in scene_classes_to_render:
+        input_file = config.get_dir("input_file")
+        if config["use_js_renderer"]:
             try:
-                if config["use_js_renderer"]:
-                    if frame_server_impl is None:
-                        raise ImportError(
-                            "Dependencies for JS renderer is not installed."
-                        )
-                    frame_server_impl.get(SceneClass).start()
-                else:
-                    scene = SceneClass()
-                    scene.render()
-                    open_file_if_needed(scene.renderer.file_writer)
+                if frame_server_impl is None:
+                    raise ImportError("Dependencies for JS renderer is not installed.")
+                server = frame_server_impl.get(input_file)
+                server.start()
+                server.wait_for_termination()
             except Exception:
                 print("\n\n")
                 traceback.print_exc()
                 print("\n\n")
+        else:
+            for SceneClass in scene_classes_from_file(input_file):
+                try:
+                    scene = SceneClass()
+                    scene.render()
+                    open_file_if_needed(scene.renderer.file_writer)
+                except Exception:
+                    print("\n\n")
+                    traceback.print_exc()
+                    print("\n\n")
```

Comment:
Why not something like
```suggestion
                except Exception as e:
                    print(e)
```

Reply:
Doing so would change the format of the error messages, which isn't the point of this PR.